api/RestfulService.php

```diff
@@ -53,7 +53,7 @@ public static function flush() {
 	 */
 	public static function set_default_curl_option($option, $value) {
 		Deprecation::notice('3.2', 'Use the "RestfulService.default_curl_options" config setting instead');
-		Config::inst()->update('RestfulService', 'default_curl_options', array($option => $value));
+		Config::inst()->update(get_called_class(), 'default_curl_options', array($option => $value));
```

Comment:
That's ok -- I just commented on that here. I don't think that's possible. Looking in my IDE, at least, I don't see an `->update()` method on the `Config_ForClass` class which is returned by the `::config()` container on `Object`.